src/frontend/web_application/src/components/ContactDetails/index.jsx

```diff
@@ -182,21 +203,62 @@ class ContactDetails extends Component {
         return this.renderAddress(detail, detailKey);
       }));
 
-    if (this.state.editMode) {
+    if (editMode.details) {
       detailKey++;
       contactDetails = contactDetails.concat(<AddressForm key={detailKey} onSubmit={onAddContactDetail} __={__} />);
     }
 
-
     return (
-      <div className="m-contact-details">
-        <Subtitle hr actions={this.renderSubtitleActions()}>
-          {__('contact.contact_details')}
-        </Subtitle>
 
-        <div className="m-contact-details__list">
-          <TextList>{contactDetails}</TextList>
+      <div className="m-contact-details">
+        <div className="m-contact-details__panel">
+          <Subtitle hr actions={this.renderSubtitleActions('details')}>
+            {__('contact.contact_details')}
+          </Subtitle>
+          <div className="m-contact-details__list">
+            <TextList>{contactDetails}</TextList>
+          </div>
         </div>
+        {contact.organizations !== {} &&
+          <div className="m-contact-details__panel">
+            <Subtitle hr actions={this.renderSubtitleActions('organizations')}>
+              {__('contact.contact_organizations')}
+            </Subtitle>
+              <div className="m-contact-details__list">
+                <TextList>
+                {contact.organizations.map(organization =>
+                    <OrgaDetails
+                      key={organization.organization_id}
+                      organization={organization}
+                      editMode={editMode.organizations}
+                      onDelete={onDeleteContactDetail}
+                      __={__} />
+                  )}
+                  </TextList>
+                {editMode.organizations && <OrgaForm onSubmit={onAddContactDetail} __={__}/>}
+              </div>
+          </div>
+        }
+        {contact.identities !== [] &&
```

Comment:
I'm not sure if identities is always defined in contacts
(API might not send props which are empty)

```
{contact.identities && contact.identities.length > 0 && ( ...
```